lib/rapidsms/tests/scripted.py

```diff
@@ -192,6 +193,11 @@ def runParsedScript (self, cmds):
                 last_msg = txt
         finally:
             self.stopRouter()
+            try:
+                transaction.commit()
+            except Exception:
```

Comment:
i was worried about it failing because there was nothing to do, in which case it shouldn't matter if it doesn't go through. but i didn't actually test to see whether that is how it works or not so happy to put in a `raise` instead if you think it's better and we can change if it fails